baselines/dom.generated.d.ts

```diff
@@ -4752,6 +4752,15 @@ declare var FileReader: {
     readonly LOADING: number;
 };
 
+interface FileReaderProgressEvent extends ProgressEvent {
+    readonly target: FileReader | null;
+}
+
+declare var FileReaderProgressEvent: {
```

Comment:
this should not be added. set `"no-interface-object": 1,` on the new interface to avoid the creation of a var.